dashboard-ui/src/components/TextBasedScenarios/TextBasedScenariosPage.jsx

```diff
@@ -59,19 +60,18 @@ class TextBasedScenariosPage extends Component {
     }
 
     introSurveyComplete = (survey) => {
-        console.log(survey.data)
         const scenarioOrderString = survey.data.scenarioOrder.replace(/\\/g, "");
         const scenarioOrderArray = JSON.parse(scenarioOrderString);
-        console.log(scenarioOrderArray)
         this.setState({
             scenarios: scenarioOrderArray,
             participantID: survey.data["Participant ID"],
             vrEnvCompleted: survey.data["vrEnvironmentsCompleted"]
         })
 
-        const selectedScenarios = []
+        let selectedScenarios = []
```

Comment:
Why was this changed to let? I don't see a reassignment of it

Reply:
I think just leftover from when I was debugging, I'll change it back